app/src/main/java/com/github/se/travelpouch/model/profile/ProfileModelView.kt

```diff
@@ -85,4 +85,15 @@ class ProfileModelView @Inject constructor(private val repository: ProfileReposi
           onFailure(it)
         })
   }
+
+  fun removeFriend(friendUid: String, onSuccess: () -> Unit, onFailure: (Exception) -> Unit) {
```

Comment:
Please add minimal documentation to this function

Reply:
Done